pages/common/jest.md

```diff
@@ -0,0 +1,23 @@
+# jest
+
+> A zero-configuration JavaScript testing platform.
+
+- Run all tests:
+
+`jest`
+
+- Run only tests with a name that matches the regex pattern:
+
+`jest -t {{spec_name}}`
+
+- Run only tests relating to uncommitted files:
+
+`jest --onlyChanged`
```

Comment:
What I really would like to do here is show both the full and shorthand versions of this option, say with `jest -o` as the second line, but couldn't find examples like this to follow.

Reply:
Yeah. We generally prefer the long options only in tldr-pages. I think the idea is that you can then look up the shortcuts as a more advanced user if you find yourself typing an option a lot.

Also, please could you avoid squashing your commits? It make it harder to see what you've changed, and we've got a special button that does it for us at merge-time :-)

Reply:
I amended the original commit since it was a direct adjustment to the feature I was adding, that has not yet reached your stable branch. It is also in my clone repository 😛.

When merged I will delete this clone anyway – so given you will squash changes on merge this history is lost anyway except in the history of this PR on GitHub.

Reply:
The point is it makes it difficult for us to review your PR.